### PR TITLE
avoid hexagon collision

### LINKED ISSUE #313: Mobile button styling
Split from #230

Android - Samsung Galaxy J600FN**
- [ ] Minor styling issue with buttons. this could be related to general background decoration in #312 
Screenshot_070319_120145_PM

## Plan
- [ ] 

## Acceptance Criteria
- [ ]

### PR DESCRIPTION
**Pull request by falconx**
Fixes #313 

## Description

* scale background hexagons with viewport
* add shadow around button controls

I've kept the hexagons large here to show the button shadow working:
hex-background

### CODE REVIEW

File: src/components/Layout/Page.js [hunk 1]
```diff
@@ -25,6 +25,9 @@ const Page = ({ title, children }) => (
         background-image: url('${hexagonsLeft}'), url('${hexagonsRight}');
         background-repeat: repeat-y;
         background-position: top left, top right;
+
+        /* scale background with viewport */
+        background-size: calc(100px + (300 - 100) * ((100vw - 320px) / (1920 - 320)));
```
Comment (RpprRoger) [inline src/components/Layout/Page.js]: clever!

Comment (falconx) [reply src/components/Layout/Page.js]: Yeah it's a cool technique based on the fluid type algorithm :slightly_smiling_face:

```
calc([minimum size] + ([maximum size] - [minimum size]) * ((100vw - [minimum viewport width]) / ([maximum viewport width] - [minimum viewport width])));
```

https://css-tricks.com/snippets/css/fluid-typography

Comment (RpprRoger) [reply src/components/Layout/Page.js]: Could you comment that in the component please?

Comment (falconx) [reply src/components/Layout/Page.js]: Also, fyi, `min`/`max` and `clamp` will make this way easier in the future :slightly_smiling_face:
https://css-tricks.com/simplified-fluid-typography

**Pull request merged**